### ISSUE TITLE
Question about css structure and scope issues

### ISSUE THREAD

**Issue by tjenkinson** [clappr/clappr#582]
Just wondering if there's a reason you decided to apply css to data tags in several places instead of classes? E.g. `data-player` has css mapped to it.

Secondly at the moment the css isn't scoped meaning if I happened to have another element on my page with `data-player` for example it inherits clappr's styles which is undesired behaviour. A simple fix for this would be to add a `clappr` class to the outer div and then prepend `.clappr` to the start of all the styles.

https://github.com/webpack/css-loader (with "CSS Modules" enabled) might be a nice solution though to help make each component in the player have its own css scope.

**Comment by towerz** [comment]
@tjenkinson we originally considered the data tags to add some scope to the markup elements and passing some information around. Since every element had data attributes, we used those in the CSS to make the selectors more specific. Some elements haven't been revisited to add classes to the data attribute selectors, but we dropped the idea of having data attributes in a later moment.

Since the adoption of webpack and css-loader, we are still reworking the scope of the css classes, but we didn't have the time yet to do it so. One of our Clappr-based players already use scoping instead of data attributes, but we still need to change this on Clappr itself.
